observers/slack_resource.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef SERENITY_SLACK_RESOURCE_HPP
+#define SERENITY_SLACK_RESOURCE_HPP
+
+#include <memory>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+#include "mesos/mesos.hpp"
+
+#include "stout/result.hpp"
+
+#include "serenity/executor_set.hpp"
+#include "serenity/serenity.hpp"
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * SlackResourceObserver observes incoming ResourceUsage
+ * and produces Resource with revocable flag set (Slack Resources).
+ *
+ * Currently it only counts CPU slack
+ */
+class SlackResourceObserver : public Consumer<ResourceUsage>,
```

Comment:
I changed the output to Producer<Resources>, but you commented the wrong line, so this discussion is not outdated :(